package.json

```diff
@@ -12,6 +12,7 @@
     "babel-plugin-react-transform": "^1.0.3",
     "css-loader": "^0.21.0",
     "del": "^1.2.0",
+    "es5-shim": "~4.1.15",
```

Comment:
There's a `karma-es6-shim`, which `peer depends` on `es5-shim` and `es6-shim`.
All that stuff is needed to **run tests in phantomjs on travis**

Note: those things are already in `yo/react`, this branch need to be rebased.